server/api/constants.go

```diff
@@ -45,4 +45,5 @@ const (
 	PropTeamID    = "team_id"
 	PropChannelID = "channel_id"
 	PropPostID    = "post_id"
+	PropUserAgent = "scope"
```

Comment:
???

Reply:
So far we have been calling that scope. We can do the change in the webapp/mobile repo and call it something different (like user_agent). Any preference on the naming?

Reply:
I don't think `scope` is intention-revealing. `user-agent` may have confusing expectations as to the format of the string. 0/5 `user-agent-type`?